src/net_processing.cpp

```diff
@@ -947,10 +947,10 @@ bool PeerManagerImpl::TipMayBeStale()
 {
     AssertLockHeld(cs_main);
     const Consensus::Params& consensusParams = m_chainparams.GetConsensus();
-    if (m_last_tip_update == 0) {
-        m_last_tip_update = GetTime();
+    if (m_last_tip_update == std::chrono::seconds::zero()) {
```

Comment:
```suggestion
    if (m_last_tip_update.count() == 0) {
```

I'd use this, but there is really no difference, so both are fine.

Reply:
Better to keep the coding usage throughout the codebase consistent.

Addressed in 8e0ba04